configure

```diff
@@ -191,8 +191,7 @@ fi
 # Check the version of libghdl is correct.
 if [ "$enable_libghdl" = true ]; then
     libghdl_version="$srcdir/python/libghdl/version.py"
-    # Compare a fixed number of bytes (-n), to avoid false positives due to LF/CRLF mismatch.
-    if ! echo "__version__ = '${ghdl_version}'" | cmp -n "${#ghdl_version}" "$libghdl_version" ; then
+    if [ "${ghdl_version}" != "$(sed "s/.*'\(.*\)'.*/\1/g" $libghdl_version)" ]; then
```

Comment:
If think you should keep the part of the comment about LF/CRLF.
I am not sure that `$(` is standard posix, if not could you use ` instead ?

Reply:
I kept it, but I removed the part about 'fixed number of bytes', which does not apply now. I also replaced `$(`.